### DISCUSSION TITLE
Is it possible to have Container Settings for each Contaciner?

### DISCUSSION THREAD

**Discussion by vnetonline** [oqtane/oqtane.framework#3295] [Q&A] [answered]
@sbwalker 

Is it possible to have Container Settings for each Container?

I currently have 4 Containers each have different settings, it would be nice to have a `ContainerSettings` for each of them

![image](https://github.com/oqtane/oqtane.framework/assets/15242923/d258181e-65bd-4247-8332-9cad37c3e011)

**Comment by sbwalker** [comment, marked as answer]
There can only be 1 ContainerSetting UI for a theme. However you could include a select option on the ContainerSetting UI which allows the user to choose from the various containers in the theme  (and default to the one currently assigned). Based on what the user chooses you can display different UI options. The options can be saved to the database with unique SettingName prefixes to differentiate them.
